File: src/main/java/com/prog5/items/DrivingLicence.java
package com.prog5.items;

public class DrivingLicence {
    public enum Category {
        A, B , C, D, E
    }
    private String owner;

    private Category category;

    public DrivingLicence(String owner, Category category) {
        this.owner = owner;
        this.category = category;
    }

    @Override
    public String toString() {
        return "\nDrivingLicence: " +
                "\nowner: " + owner +
                ", category: " + category;
    }
}

File: DrivingLicence.cs
namespace Items 
{
  public class DrivingLicence
  {
    public string Owner { get; set; }
    public string Category { get; set; }
    
    public DrivingLicence(string owner, string category)
    {
        Owner = owner;
        Category = category;
    }
  }
}